src/interpreter.cc

```diff
@@ -94,9 +94,9 @@ FuncSignature::FuncSignature(Index param_count,
       result_types(result_types, result_types + result_count) {}
 
 Import::Import() : kind(ExternalKind::Func) {
-  WABT_ZERO_MEMORY(module_name);
-  WABT_ZERO_MEMORY(field_name);
-  WABT_ZERO_MEMORY(func.sig_index);
+  ZeroMemory(module_name);
```

Comment:
Shouldn't we have StringSlice (an maybe other struct's) initialize itself, zeroing them manually doesn't seem very robust.